### PR TITLE
A couple of perl packages for IMAP

### PR DESCRIPTION
**Pull request by oxij**

### CODE REVIEW

File: pkgs/top-level/perl-packages.nix [hunk 1]
```diff
@@ -4674,6 +4674,16 @@ let self = _self // overrides; _self = with self; {
     };
   };
 
+  IMAPClient = buildPerlPackage {
+    name = "IMAPClient-0.13";
```
Comment (edolstra) [inline pkgs/top-level/perl-packages.nix]: This should be `IMAP-Client`.

File: pkgs/top-level/perl-packages.nix [hunk 2]
```diff
@@ -10504,6 +10514,15 @@ let self = _self // overrides; _self = with self; {
     buildInputs = [URI TestMore];
   };
 
+  URIIMAP = buildPerlPackage {
+    name = "URIIMAP-1.01";
```
Comment (edolstra) [inline pkgs/top-level/perl-packages.nix]: This should be `URI-imap-1.01`.

Comment (oxij) [inline pkgs/top-level/perl-packages.nix]: Should I change the attrset name too then?

**Comment by oxij** [comment]
@edolstra should be fixed

**Pull request merged**